apps/keycloak/disable-ssl-for-master-realm.sh

```
@@ -0,0 +1,2 @@
+/opt/keycloak/bin/kcadm.sh config credentials --server http://localhost:8080 --realm master --user admin --password admin
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)